src/Extension.ts

```diff
@@ -70,5 +70,34 @@ export class Extension {
 				}
 			)
 		);
+
+		this.dispose.track(
+			vscode.commands.registerCommand(
+				"hediet.vscode-drawio.newDiagram", () => {
+					const options: vscode.SaveDialogOptions = {
+						saveLabel: 'Create',
+						filters: {
+							'Diagrams': ['drawio']
+						}
+					};
+
+					vscode.window.showSaveDialog(options).then(fileUri => {
+						if (fileUri) {
+							var fs = require('fs');
+							var emptyContent = `<mxfile version="13.7.9"><diagram name="Page-1">ddHLEoIgFAbgp2GPUE2tzWrTykVrRk7CDHoYpNF6+nSQjLFWwMcPhwvheTOcnbDqihIMYVQOhB8JY9mG7cZmkmeQA6UBaqflHFqg1C+YMcYeWkKXBD2i8dqmWGHbQuUTE85hn8buaNKqVtSwgrISZq03Lb0Kut/SxS+gaxUrZ/F+jYjhGTolJPZfxAvCc4foQ68ZcjDT48V3CetOf2Y/B3PQ+h8Lxs6y9zhIfogXbw==</diagram></mxfile>`;
```

Comment:
Please use `const` ;)
Also, you don't need to initialize the file with some content. Draw.io will do it automatically when opening empty file!

Reply:
I see. I will leave the content as empty 😄